clever.wdl

```diff
@@ -0,0 +1,82 @@
+version 1.0 
+
+import "common.wdl"
+import "bwa.wdl"
+task Prediction {
+    input {
+        File bamFile
+        File bamIndex
+        BwaIndex bwaIndex
+        String outputPath        
+        Int threads = 10 
+        Int mem = 15
+        String dockerImage = "quay.io/biocontainers/clever-toolkit:2.4--py36hcfe0e84_6"
+    }   
+    
+
```

Comment:
```suggestion
```

Reply:
I thought we keep a default docker-image in each task?

Reply:
We do. I think @JasperBoom just meant that the extra blank line should be removed

Reply:
Ah ok! got it